spec/message.ebnf

```diff
@@ -1,11 +1,11 @@
-Message ::= Plain | Pattern | Preamble Variant+
+Message ::= Declaration* ( Pattern | Selector Variant+ )
 
 /* Preamble */
-Preamble ::= Selector+
-Selector ::= (Variable '=')? '{' Expression '}'
+Declaration ::= 'let' WhiteSpace Variable '=' '{' Expression '}'
```

Comment:
I don’t think the EBNF requires that we be explicit about white space here.

```suggestion
Declaration ::= 'let' Variable '=' '{' Expression '}'
```

Reply:
Actually, that's wrong. Sorry for the confusion! It's the other way around: we need `/* ws: explicit */` when we want to forbid _optional_ whitespace, not when we want to require at least some. I remembered wrong.